docs/contributing/contributor-compensation.md

```diff
@@ -2,29 +2,42 @@
 
 ---
 
-## <img class="dcr-icon" src="/img/dcr-icons/RFP.svg" /> Request for Proposals (RFP)
+## <img class="dcr-icon" src="/img/dcr-icons/User.svg" /> Contractors
 
-Decred was launched with a Request For Proposal system in order to provide compensation for contributors working on larger or more significant projects. A document (RFP) would be produced which described the requirements and scope of a project, along with a clearly defined set of milestones and a reward specified in DCR. The RFP would be posted publicly and community members would be given a period of time to submit their proposals. After review, a proposal would be selected and awarded the contract.
+The term "Contractors" covers both **corporate contractors** and **independent contractors**.
 
-The RFPs were stored on GitHub and they can still be viewed. The first, [RFP-001](https://github.com/decred/RFPs/blob/master/rfp-0001/rfp-0001.md), was looking for developers to overhaul the Paymetheus application. RFP progress and payments were made public via the Decred forum[^1].
+* Corporate contractors are companies that do work for Decred, e.g. Company 0, Raedah, Ditto. Corporate contractors typically employ a number of contributors who work on Decred (but for the corporate contractor).
+* Independent contractors are people that do work for Decred. [This page](contributing-to-decred.md) explains how people who contribute to the project can become independent contractors.
 
-The RFP process worked well for certain types of tasks, however a need arose to have contributors engaged and compensated over the longer term and on a more flexible basis[^2]. In January 2017 the RFP system was changed to a contractor model and Decred began to hire it's first contractors.
+Contractors contribute in a wide variety of ways including software development, design, communications, community support, research and documentation.
+ 
+Decred currently has over 50 contributors (independant contractors and employees of corporate contractors), and you can see them on [decred.org/contributors](https://decred.org/contributors/) in portraits with badges indicating whether they work for a corporate contractor.
 
 ---
 
-## <img class="dcr-icon" src="/img/dcr-icons/User.svg" /> Contractors
+## <img class="dcr-icon" src="/img/dcr-icons/Wallet.svg" /> Billing and payments
+
+At the end of each month, each contractor prepares an invoice and submits it to the _Decred Holdings Group_ LLC (DHG). The invoice lists the items the contractor has worked on and a charge (in $USD) for each - typically hours worked charged at an agreed hourly rate.
```

Comment:
1) drop italics for DHG; 2) "hours worked are charged"; 3) is `$USD` a common notion for currencies (vs just `USD`)?